packages/app/src/cli/utilities/extensions/cli.ts

```diff
@@ -19,35 +15,18 @@ const NodeExtensionsCLINotFoundError = () => {
  */
 export async function runGoExtensionsCLI(args: string[], options: system.WritableExecOptions = {}) {
   const stdout = options.stdout || {write: () => {}}
-  if (useExtensionsCLISources()) {
-    const projectDirectory = path.join(
-      environment.local.homeDirectory(),
-      'src/github.com/shopify/shopify-cli-extensions',
-    )
-    stdout.write(`Using extensions CLI from ${projectDirectory}`)
+  if (environment.local.isDebug()) {
+    const extensionsGoCliDirectory = (await path.findUp('packages/ui-extensions-go-cli/', {
```

Comment:
would it make sense to store this kind of paths to packages in a utility, something like a directory resolver for packages?

Reply:
Good point. I will create an issue to add that feature.